src/sentry/notifications/notifications/rules.py

```diff
@@ -180,6 +179,8 @@ def __init__(
         target_identifier: int | None = None,
         last_release: Optional[Release] = None,
     ) -> None:
+        from sentry.rules.conditions.active_release import ActiveReleaseEventCondition
```

Comment:
ideally we should split the part of the code which is causing it to be circular to a separate module rather than local imports

Reply:
Yeah, I was thinking about that as well. But this class is part of an experiment and I expect this to be throw-away code. I wanted to isolate to as few files as possible so it is easier to remove later on.